dbms/src/AggregateFunctions/ReservoirSampler.h

```diff
@@ -108,9 +108,12 @@ class ReservoirSampler
       */
     double quantileInterpolated(double level)
     {
-        if (samples.empty())
+        if (samples.empty()) {
+            if (DB::IsDecimalNumber<T>) {
```

Comment:
the same: no braces here.

Reply:
code style fixed
I started using check-style utility